crest/Assets/Crest/Crest/Scripts/Spline/Spline.cs

```diff
@@ -29,6 +29,16 @@ public partial class Spline : MonoBehaviour, ISplinePointCustomDataSetup
         int _version = 0;
 #pragma warning restore 414
 
+        public enum Offset
+        {
+            Left,
+            Center,
+            Right
+        }
+        [Tooltip("Where generated ribbon should lie relative to spline. If set to Center, ribbon is centered around spline.")]
+        // 01-Oct-21: Default to right for now to support legacy data. In the future we should update this default to Center.
```

Comment:
Could use the ISerializationCallbackReceiver and set version to one similar to clip surface primitives? Details: https://github.com/cowhunt/crest-wiki-private/wiki/Data-Upgrade. Being an MB means it should work fine.

Reply:
yeah that works, and i guess is more explicit (and avoids future todo) so ive gone with that

Reply:
Thanks.